README.md

```diff
@@ -33,51 +33,48 @@ boards: # All boards on the car, along with CAN messages they publish and subscr
       can_bus0:
         - Msg0
 buses: # List of CAN buses on the car
-  can_bus0: # All messages on can_bus0
+  can0: # All messages on can0
     baudrate: 500000
-    is_extended: false
+    extended: false
     messages:
       Msg0:
-        can_id: 0x001 # Note that lower CAN IDs have higher priority
-        frequency: 50Hz # Frequency is optional, only add it if you will use it
-        is_big_endian: true
+        id: 0x001 # Note that lower CAN IDs have higher priority
+        period: 50ms # optional, only add it if you will use it
         segments:
           segment0:
-            position: 0 # Which bit in the CAN message the segment starts at
-            length: 32 # How many bits long the segment is
-            c_type: int32_t
-            signed: true # The signed field is technically optional, but good to add for integers
-            unit: V # The unit field is also optional
+            slice: 0 + 32  # [start index] + [length]
```

Comment:
Why do you use pluses instead of commas?

Reply:
Pluses make more sense when you want to do `+ 32` instead of having to manually type the starting position. This is going to be a good feature.

Reply:
Ah I see. I mostly just saw this first one and misinterpreted it. Sounds good.